Reaktoro/ODML/PriorityQueue.cpp

```diff
@@ -76,15 +75,22 @@ auto PriorityQueue::reset() -> void
     std::iota(_order.begin(), _order.end(), 0);
 }
 
-auto PriorityQueue::increment(Index ientity) -> void
+auto PriorityQueue::increment(Index identity) -> void
 {
     // == EXAMPLE OF WHAT HAPPENS IN THIS METHOD ==
     // PRIORITIES BEFORE INCREMENTING: 13  5  3  2  2 (2) 1  --- incrementing from 2 to 3
-    //  PRIORITIES AFTER INCREMENTING: 13  5  3 [2] 2 (3) 1  --- (3) needs to be swaped with [2]
+    //  PRIORITIES AFTER INCREMENTING: 13  5  3 [2] 2 (3) 1  --- (3) needs to be swapped with [2]
     //       PRIORITIES AFTER SORTING: 13  5  3 (3) 2 [2] 1
-    _priorities[ientity] += 1;
-    std::sort(_order.begin(), _order.begin() + ientity + 1,
-        [&](Index l, Index r) { return _priorities[l] > _priorities[r]; });
+    _priorities[identity] += 1;
+
+    // Avoid sorting if the identity == 0 (the priority of the very first element is increased)
```

Comment:
It would be nice to have an example here in comments to explain this logic (something similar to the comment above with title `== EXAMPLE OF WHAT HAPPENS IN THIS METHOD ==`).

Reply:
Added